src/app/views/cheat-sheets/basic-power/basic-power.component.html

```diff
@@ -42,19 +42,19 @@ <h4 class="card-title" id="steam-power">Steam Power</h4>
     <ul>
         <li>
             An <a href="https://wiki.factorio.com/Offshore_pump" target="_blank" rel="noopener">offshore pump</a>
-            provides {{ sheetData?.offshore_pump_water }} water per second.
+            provides {{ sheetData?.offshore_pump_water }} units of water per second.
         </li>
         <li>
             A <a href="https://wiki.factorio.com/Boiler" target="_blank" rel="noopener">steam boiler</a>
-            can turn up to {{ sheetData?.steam_boiler_fluid }} water units to steam per second,
-            providing {{ sheetData?.steam_boiler_energy }} of steam (at 165C).
+            can convert up to {{ sheetData?.steam_boiler_fluid }} units of water into steam per second,
+            providing {{ sheetData?.steam_boiler_energy }} of steam (at 165 °C).
```

Comment:
°C - Fancy :p

Reply:
Some part of the guide did already use °C though... I don't remember where that was.
So now we're consistent the whole way through.